src/question/order_select/builder.rs

```diff
@@ -0,0 +1,280 @@
+use ui::backend::Backend;
+use ui::widgets::Text;
+
+use crate::{
+    question::{Choice, Options},
+    ListItem,
+};
+
+use super::OrderSelect;
+
+/// Prompt that allows the user to organize a list of options.
+///
+/// The choices are represented with the [`Choice`] enum. [`Choice::Choice`] can be multi-line.
+/// but [`Choice::Separator`]s will be ignored.
+///
+/// // TODO : add gif
+/// <img
+///   src="https://raw.githubusercontent.com/lutetium-vanadium/requestty/master/assets/multi-select.gif"
+///   style="max-height: 20rem"
+/// />
+///
+/// See the various methods for more details on each available option.
+///
+/// # Examples
+///
+/// ```
+/// let order_select = Question::order_select("tasks")
+///     .message("Please organize the tasks")
+///     .choices(vec![
+///         "Task 1",
+///         "Task 2",
+///         "Task 3",
+///     ])
+///     .build();
+/// ```
+///
+///
+/// [`order_select`]: crate::question::Question::order_select
+#[derive(Debug)]
+pub struct OrderSelectBuilder<'a> {
+    opts: Options<'a>,
+    order_select: OrderSelect<'a>,
+}
+
+impl<'a> OrderSelectBuilder<'a> {
```

Comment:
Since they're meant to indicate the user that there is no specific order needed in the options, i was thinking about turning them into comments instead of deleting them

What do you think about that ?

Reply:
Yup that works too